cocos/spine/lib/spine-define.ts

```diff
@@ -372,173 +176,38 @@ function overrideProperty_RegionAttachment (): void {
     });
 }
 
-function overrideProperty_TextureAtlas (): void {
-    // const prototype = spine.TextureAtlas.prototype as any;
-    // const propertyPolyfills = [
-    //     {
-    //         proto: prototype,
-    //         property: 'pages',
-    //         getter: prototype.getProp_pages,
-    //     },
-    //     {
-    //         proto: prototype,
-    //         property: 'regions',
-    //         getter: prototype.getProp_regions,
-    //     },
-    // ];
-    // propertyPolyfills.forEach((prop) => {
-    //     js.getset(prototype, prop[0], prop[1]);
-    // });
-}
-
-function overrideProperty_SlotData (): void {
-    const prototype = spine.SlotData.prototype as any;
-    const propertyPolyfills = [
-        ['index', prototype.getIndex],
-        ['boneData', prototype.getBoneData],
-        ['name', prototype.getName],
-        ['color', prototype.getColor],
-        ['darkColor', prototype.getDarkColor],
-        ['blendMode', prototype.getBlendMode, prototype.setBlendMode],
-    ];
-    propertyPolyfills.forEach((prop): void => {
-        js.getset(prototype, prop[0], prop[1], prop[2]);
-    });
-}
-
 function overrideProperty_IkConstraint (): void {
     const prototype = spine.IkConstraint.prototype as any;
-    const propertyPolyfills = [
-        ['data', prototype.getData],
-        ['target', prototype.getTarget, prototype.setTarget],
-        ['bendDirection', prototype.getBendDirection, prototype.setBendDirection],
-        ['compress', prototype.getCompress, prototype.setCompress],
-        ['stretch', prototype.getStretch, prototype.setStretch],
-        ['mix', prototype.getMix, prototype.setMix],
-        ['softness', prototype.getSoftness, prototype.setSoftness],
-        ['active', prototype.getActive, prototype.setActive],
-    ];
-    propertyPolyfills.forEach((prop): void => {
-        js.getset(prototype, prop[0], prop[1], prop[2]);
-    });
```

Comment:
Remove lots of `property` polyfill code in spine-define.ts since we could use `.property(...)` to bind them.